packages/testing/src/dispose.ts

```diff
@@ -23,8 +23,12 @@ export const DEFAULT_DISPOSAL_GROUP = DEFAULT_GROUP;
  *
  * @param group disposal group name. disposal groups let you specify disposal constrains. see: {@link createDisposalGroup}
  */
-export function disposeAfter(disposable: DisposableItem, options?: DisposableOptions) {
-    disposables.add(disposable, options);
+export function disposeAfter(disposable: DisposableItem, options: string | Omit<DisposableOptions, 'dispose'>) {
```

Comment:
it would be nice to have the same signature over here as well.
also the jsdocs seems a bit out-of-touch even before this change